lib/Module.js

```diff
@@ -69,7 +69,7 @@ class Module extends DependenciesBlock {
 		this._chunks = new SortableSet(undefined, sortById);
 
 		// Info from Compilation (per Compilation)
-		/** @type {number | string} */
+		/** @type {number|string} */
```

Comment:
This is bugging me. `null` is neither a `number` nor a `string` but it definitely gets initialized right after the module is instantiated. I really hope we won't get bite by a typing issue in the future because of this.

Reply:
Yup! With strictNullCheck's infact, this errors instantly. To be sound we either have to (throughout entire reference of module), for id, have null checks or instead instantly set it to a number.